saic-java-mqtt-gateway/src/main/java/net/heberling/ismart/mqtt/VehicleState.java

```diff
@@ -709,6 +730,17 @@ public void configure(String topic, MqttMessage message) {
           throw new MqttGatewayException("Error setting value for payload: " + message);
         }
         break;
+      case CLIMATE_REMOTE_TEMPERATURE:
+        try {
+          int temperature = Integer.parseInt(message.toString());
+          if(temperature < 16 || temperature > 28) {
```

Comment:
This should go into the setter, and must probably be car specific. I have seen different min/max values in the different app versions

Reply:
yep, also first test. This is a quite interesting idea though, we have follwing stuff that is car specific:
* Mapping
* Min / Max 
* Steps (in case of °C/Farenheit)

I am wondering if it is such a good idea to try handle all this complexity in the car, or if having a default behaviour plus a chance to inject a config file that every user can configure on his own?